django/core/cache/backends/memcached.py

```diff
@@ -194,6 +194,56 @@ def __init__(self, server, params):
     def _cache(self):
         return self._lib.Client(self._servers, **self._options)
 
+    def add(self, key, value, timeout=DEFAULT_TIMEOUT, version=None):
```

Comment:
Since the args/kwargs aren't used directly, would this be simpler to maintain as `*args, **kwargs` instead? (I can just see a case where it ends up out of sync with `BaseMemcachedCache`)

Reply:
Oh `value` is used, I can't read! (There are some methods that don't use the passed arguments, but perhaps they should all be specified for consistency)